clang/lib/CodeGen/CoverageMappingGen.cpp

```diff
@@ -174,6 +179,10 @@ class SourceMappingRegion {
 
   void setGap(bool Gap) { GapRegion = Gap; }
 
+  bool isSkipped() const { return SkippedRegion; }
```

Comment:
Unrelated to this patch: I wonder if `SkippedRegion` is being used elsewhere. This could be nice clean up.

Reply:
SkippedRegions are also created by preprocessor for comments and whitespaces, then they are squeezed out of line with any coverage. This patch allows making them also while walking AST and fixes `LineCoverageStats` to properly show such lines as mapped.